packages/time-series/lib/commands/ADD.spec.ts

```diff
@@ -57,16 +57,44 @@ describe('TS.ADD', () => {
       );
     });
 
-    it('with RETENTION, ENCODING, CHUNK_SIZE, ON_DUPLICATE, LABELS', () => {
+    it('with IGNORE no values', () => {
+      assert.deepEqual(
+        ADD.transformArguments('key', '*', 1, {
+          IGNORE: { }
```

Comment:
Is "IGNORE no values" supported? (https://github.com/RedisTimeSeries/RedisTimeSeries/pull/1543)
If it is - add support for `IGNORE: true` for this case